index_V1

```diff
@@ -29,49 +69,141 @@
+						
+							<li style="margin-right: 0vw;"><a href="login.php">Sign Out</a></li>
+						
+						<?php else : ?>
+							<li style="margin-right: -1vw;" class="account"><a href="">
+								
+								<img src="images/icon_4.png" style="height: 1.95vw; width: 1.95vw; margin-right: 1vw; margin-bottom: -0.4vw;" draggable="false">
+								<h3 style="font-size: 1.3vw; float: right; margin-top: -0.1vw;"><?php echo $User ?></h3>
+							
+							</a></li>
+						
+							<li style="margin-right: -10vw;"><a href="login.php">Sign Out</a></li>
+							<style>
+								ul li {
+									margin-right: 55vw;
+								}
+								.account:hover {
+									background: transparent;
+									transform: scale(1.02, 1.02);
+									transition: 0.6s;
+								}
+							</style>
+						<?php endif; ?>
+					</ul>
+				</div>
+					
+				
+				<div class="scroll-area">
+					
+					<h1>Placeholder.</h1>
+					<h2>Lorem Ipsum.</h2>
+				
+				</div>
+				
+				<div class="bar">
+				
+					<form method = "post" style="margin-top: 1vw;">
+						
+					  <input type="text" autocomplete="off" class="search" placeholder="Search..." name="searched">
+						
+					  <button class="button" type="submit" name="search" style="height: 3vw; width: 6vw; background-color: grey; float: right; margin-top: -2.9vw;"><img src="images/icon_6.png" style="width: 1.2vw; height: 1.2vw;"></button>
+						
+					</form>
+				
+				</div>
+			
+			</div>
 			
 			<div class="circles">
 				
 				<div class="table">
 					
-					<div onclick="location.href='query_1.php';" class="circle_1" style="cursor: pointer; grid-column: 1; background-color: #e6e6e6; border-radius: 30vw; margin: 0vw 1.5vw; transition: 0.6s;"><p style="margin: 15vw 3.5vw; transition: 0.6s;">Put Link Here</p></div>
+					<div onclick="location.href='query_1.php';" class="circle_1" style="cursor: pointer; grid-column: 1; background-color: #e6e6e6; overflow: hidden; border-radius: 1.5vw; margin: 0vw 1.5vw; transition: 0.6s;">
+						
+						<img src="images/img_8.jpg" img draggable="false" alt="Blue" style="width: 32vw; height: 16vw;">
 					
-					<div onclick="location.href='query_1.php';"class="circle_2" style="cursor: pointer; grid-column: 2; background-color: #e6e6e6; border-radius: 30vw; margin: 0vw 1.5vw; transition: 0.6s;"><p style="margin: 15vw 3.5vw; transition: 0.6s;">Put Link Here</p></div>
+					</div>
 					
-					<div onclick="location.href='query_1.php';"class="circle_3" style="cursor: pointer; grid-column: 3; background-color: #e6e6e6; border-radius: 30vw; margin: 0vw 1.5vw; transition: 0.6s;"><p style="margin: 15vw 3.5vw; transition: 0.6s;">Put Link Here</p></div>
+					<div onclick="location.href='query_2.php';"class="circle_2" style="cursor: pointer; grid-column: 2; background-color: #e6e6e6; overflow: hidden; border-radius: 1.5vw; margin: 0vw 1.5vw; transition: 0.6s;">
+						
+						<img src="images/img_9.jpg" img draggable="false" style="width: 32vw; height: 16vw;" alt="Blue">
```

Comment:
This little code makes it so that image used in the circle one cannot be draggable and is fixed to the circle div.